File: strings/notes.py
#Maisha Paredes, String notes python

#String is a data type that holds information surrounded by quotation marks like: "" ''

#note = "Teachers class"

#name = input("What is your first name?").strip().upper()

#print(f"Hello {name} welcome to my program!")

#print("this is your name "+ name)

sentence = "The quick brown fox jumps over the lazy dog"

#print(len(sentence))

#print(sentence[4])
start = sentence.find("brown")
length = len("brown fox")
print(sentence[start:start+10:19])

File: strings/notes.c
// Maisha Paredes, notes c
#include <stdio.h>
#include <string.h>

char subject[50];
char name[] = "Victoria";\
char sentence[] = "The quick brown fox jumps over the lazy dog."

int main(void){
   //printf("What class are you in?");
   //scanf("%s", subject);
   //fgets(subject, sizeof(subject), stdin);
   //printf("Dude, your in %s, ? that is a COOL class!", subject);

   //printf("%s", name);
   //name[0] = "T";;
   //name[1] = "o";
   //name[2] = "r";
   //name[3] = "i";
   //printf("%lu\n", sizeof(sentence));
   //printf(strlen(sentence));

char one[] = "Hello";

char two[] = "World";

char three[] = "Welcome to my program.";

printf("%s\n", one);
strcat(one, two);
printf("%s\n", one);
strcat(three, one);

return 0;
    }